src/services/doppler-api-client.ts

```diff
@@ -163,6 +164,7 @@ export class HttpDopplerApiClient implements DopplerApiClient {
       campaignSubject: x.campaignSubject,
       clicksCount: x.clicksCount,
       deliveryStatus: x.deliveryStatus,
+      isSendingNow: x.isSendingNow,
```

Comment:
Beware here, right now api is not deployed, so I guess this value will be undefined. Did you take it in account?

Reply:
Yes, this value will be undefined and it will use the reports url because of this validation:

![image](https://user-images.githubusercontent.com/10213170/91894118-8b3e6780-ec6b-11ea-9d21-b381f27a9acb.png)